src/mvp/MvpServer.cpp

```diff
@@ -113,10 +113,14 @@ void MvpServer::ProcessTurn() {
 
     // Give coins every second
     if (m_turn % TICK_RATE == 0) {
-        m_playerTop.coins += 10;
-        m_playerBottom.coins += 10;
+        m_playerTop.coins += 2;
+        m_playerBottom.coins += 2;
```

Comment:
Maybe we can define a constant named something like `REWARD_PASSIVE` that say show many coins are rewarded passively.